tests/integration_test.rs

```diff
@@ -0,0 +1,55 @@
+#[cfg(test)]
+use assert_cmd::Command;
+
+#[test]
+fn deploy_and_invoke_contract_against_rpc_server() {
+    // This test assumes a fresh standalone network rpc server on port 8000
+
+    let mut cmd = Command::cargo_bin("soroban").unwrap();
+    // TODO: How to reuse the env variables between deploy and invoke
```

Comment:
@leighmcculloch I would love your input on this. I tried creating a factory function building a base command but it didn't work since I cannot derreference the command and return it.

Reply:
I think this should work:
```rust
fn cmd() -> Command {
    let mut cmd = Command::cargo_bin("soroban").unwrap();
    cmd.env("SOROBAN_RPC_URL", "http://localhost:8000/soroban/rpc")
        .env(
            "SOROBAN_SECRET_KEY",
            "SC5O7VZUXDJ6JBDSZ74DSERXL7W3Y5LTOAMRF7RQRL3TAGAPS7LUVG3L",
        )
        .env(
            "SOROBAN_NETWORK_PASSPHRASE",
            "Standalone Network ; February 2017",
        );
    cmd
}

#[test]
fn deploy_and_invoke_contract_against_rpc_server() {
    let mut deploy = cmd();
    let deploy = deploy
        .args(&[
            "deploy",
            "--wasm=tests/fixtures/soroban_hello_world_contract.wasm",
            "--salt=0",
        ]);

   // ...
```

Another option is to provide the list of envs:
```rust
const COMMAND_ENVS: &[(&str, &str)] = &[
    ("SOROBAN_RPC_URL", "http://localhost:8000/soroban/rpc"),
    (
        "SOROBAN_SECRET_KEY",
        "SC5O7VZUXDJ6JBDSZ74DSERXL7W3Y5LTOAMRF7RQRL3TAGAPS7LUVG3L",
    ),
    (
        "SOROBAN_NETWORK_PASSPHRASE",
        "Standalone Network ; February 2017",
    ),
];

#[test]
fn deploy_and_invoke_contract_against_rpc_server() {
    let mut cmd = Command::cargo_bin("soroban").unwrap();
    let deploy = cmd
        .envs(COMMAND_ENVS.iter().copied())
        .args(&[
            "deploy",
            "--wasm=tests/fixtures/soroban_hello_world_contract.wasm",
            "--salt=0",
        ]);

   // ...
```

A factory is simpler imo, but just mentioning the alternative in case it is preferred.

Reply:
Ahh, simple enough, thanks! I didn't manage to return a `Command` 🤷